flutter_moviehub/lib/widgets/list/showcase_list_view.dart

```diff
@@ -114,14 +117,14 @@ class ShowcaseListViewState extends State<ShowcaseListView> {
           ),
           Row(
             children: <Widget>[
-              FlatButton.icon(
-                padding: EdgeInsets.all(0),
+              TextButton.icon(
+                // padding: EdgeInsets.all(0),
                 icon: Icon(
                   Icons.timer,
                   color: Colors.white,
                   size: 16,
                 ),
-                color: Colors.transparent,
+                // color: Colors.transparent,
```

Comment:
Can we remove the unnecessary commented properties? Includes other similar changes too.